client/src/components/stream/Stream.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+class Stream extends Component{
+    /*Upon mounting of stream, the user's video, and audio will be obtained
```

Comment:
See Other Components To Declare State. You Set State On Line 23 For [videoSrc], However Compononet Level State Is Not Defined And [videoSrc] Does Not Exist.